### PR TITLE
feat: [Home] Add logged-in rails

### PR DESCRIPTION
**Pull request by damassi**
Addresses
- https://artsyproduct.atlassian.net/browse/GRO-525

This PR adds logged in rails, and organizes things per the comps. 

loggedin

### CODE REVIEW

File: src/v2/Apps/Home/Components/HomeArtworkModuleRail.tsx [hunk 1]
```diff
@@ -24,8 +24,8 @@ const HOME_ARTWORK_MODULES = [
   "FOLLOWED_ARTIST",
   "FOLLOWED_ARTISTS",
   "FOLLOWED_GALLERIES",
-  "GENERIC_GENES",
-  "FOLLOWED_GENES",
+  "GENERIC_GENES", // GENERIC_GENES
+  "FOLLOWED_GENES", // FOLLOWED_GENES
```
Comment (dzucconi) [inline src/v2/Apps/Home/Components/HomeArtworkModuleRail.tsx:28]: What's with the comments?

File: src/v2/Apps/Home/HomeApp.tsx [hunk 2]
```diff
@@ -49,16 +50,26 @@ export const HomeApp: React.FC<HomeAppProps> = ({
...
+
         <HomeFeaturedShowsRailQueryRenderer />
+
         <HomeCurrentFairsRailQueryRenderer />
+
+        <HomeFeaturedGalleriesRailQueryRenderer />
+
         <HomeTrendingArtistsRailQueryRenderer />
+
+        {/* TODO: Remove this code once we're 100% sure we don't want anymore  */}
```
Comment (dzucconi) [inline src/v2/Apps/Home/HomeApp.tsx:69]: 😁

File: src/v2/Apps/Home/Components/HomeRecentlyViewedRail.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+          <Box width={325} key={i}>
+            <SkeletonBox width={325} height={230} />
+            <SkeletonText variant="lg">Some Artist</SkeletonText>
+            <SkeletonText variant="md">Location</SkeletonText>
+          </Box>
+        )
+      })}
+    </Shelf>
+  </Skeleton>
+)
```
Comment (dzucconi) [inline src/v2/Apps/Home/Components/HomeRecentlyViewedRail.tsx:54]: It looks like this placeholder was copied over from a different style rail, doesn't match artwork shelves

File: src/v2/Apps/Home/Components/HomeWorksByArtistsYouFollowRail.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+          <Box width={325} key={i}>
+            <SkeletonBox width={325} height={230} />
+            <SkeletonText variant="lg">Some Artist</SkeletonText>
+            <SkeletonText variant="md">Location</SkeletonText>
+          </Box>
+        )
+      })}
+    </Shelf>
+  </Skeleton>
+)
```
Comment (dzucconi) [inline src/v2/Apps/Home/Components/HomeWorksByArtistsYouFollowRail.tsx:54]: Ditto

**Review by dzucconi — APPROVED** [review]
Looks good aside from the mismatched placeholders

File: src/v2/Apps/Home/Components/HomeArtworkModuleRail.tsx (hunk 1 above)
Comment (damassi) [reply src/v2/Apps/Home/Components/HomeArtworkModuleRail.tsx:28]: looks like holdovers from before I updated the types in MP; will fix

File: src/v2/Apps/Home/Components/HomeRecentlyViewedRail.tsx (hunk 3 above)
Comment (damassi) [reply src/v2/Apps/Home/Components/HomeRecentlyViewedRail.tsx:66]: Fixed

**Pull request merged**